sdk/translation/azure-ai-translation-document/tsp-location.yaml

```diff
@@ -1,3 +1,3 @@
 directory: specification/translation/Azure.AI.DocumentTranslation
-commit: 27a9398801386caaba2df7e1a4d1a8abd19e3789
-repo: Azure/azure-rest-api-specs
+commit: a2b9e584b3c6aafa1d3a977b21b0d024f28cfe1c
+repo: hamshavathimunibyraiah/azure-rest-api-specs
```

Comment:
Isnt this commit from the Azure/azure-rest-api-specs repo?

Reply:
I think https://github.com/Azure/azure-rest-api-specs/pull/30970 works. Maybe your configure here is not correct.

Mine test https://github.com/Azure/azure-sdk-for-java/pull/42338/files

Reply:
didn't think it mattered, but updated